docker/nginx.conf

```diff
@@ -0,0 +1,48 @@
+worker_processes 1;
+
+user abc abc;
+error_log /var/log/nginx/error.log warn;
```

Comment:
I think in official nginx image `/var/log/nginx/error.log` is a symlink to stdout.

It this the case for this image?